graphics/Coin/Portfile

```diff
@@ -1,80 +1,65 @@
-                    structures to render real-time graphics suitable for mostly all   \
-                    kinds of scientific and engineering visualization applications.
+PortGroup          cmake 1.1
+PortGroup          github 1.0
 
-conflicts           Coin-framework
-categories          graphics devel
-platforms           darwin
-maintainers         nomaintainer
+name               Coin
+version            4.0.0
+categories         graphics devel
+platforms          darwin
+revision           8
+maintainers        nomaintainer
+license            BSD
+description        cross platform C++ OpenGL scene graph library
+conflicts          Coin-framework
 
-homepage            https://bitbucket.org/Coin3D/coin/wiki/Home
+long_description   Coin is a high-level 3D graphics library with a \
+                   C++ Application Programming Interface. Coin uses scene-graph data \
+                   structures to render real-time graphics suitable for mostly all \
+                   kinds of scientific and engineering visualization applications.
 
-bitbucket.tarball_from downloads
-distname            ${name}-${version}
+# tag_prefix will be changed to v starting with version 4.0.1
+github.setup       coin3d coin ${version} Coin-
+github.tarball_from releases
 
-checksums           rmd160  e57fd6a4876a06874429f9ad6b96f166f5609362 \
-                    sha256  583478c581317862aa03a19f14c527c3888478a06284b9a46a0155fa5886d417
+homepage           https://coin3d.github.io
 
-depends_lib-append  port:simage \
-                    port:fontconfig \
-                    port:freetype \
-                    port:zlib \
-                    port:bzip2
+distname           coin-${version}-src
+distfiles          coin-${version}-src${extract.suffix}
 
-patchfiles          configure.patch \
-                    for.patch \
-                    clang-name-lookup.diff \
-                    fix-weird-error.diff \
-                    patch-include-Inventor-C-base-math-undefs.h.diff \
-                    patch-Info.plist.in.diff
+checksums          rmd160  29466d5cb52cf3d2cbff500d01891c9513c794c3 \
+                   sha256  e4f4bd57804b8ed0e017424ad2e45c112912a928b83f86c89963df9015251476 \
+                   size    6824162
 
-configure.args-append \
-                    --disable-debug \
-                    --disable-symbols \
-                    --disable-dependency-tracking \
-                    --without-framework
+configure.args-append   -DUSE_EXTERNAL_EXPAT=ON \
+                        -DHAVE_MULTIPLE_VERSION=ON
 
-post-build {
-    reinplace -E {s|-arch [a-z0-9_]+||g} \
-        ${worksrcpath}/Coin.pc \
-        ${worksrcpath}/coin-default.cfg
-}
+depends_lib-append port:boost \
+                   port:fontconfig \
+                   port:freetype \
+                   port:zlib \
+                   port:bzip2 \
+                   port:expat \
+                   port:simage
```

Comment:
please sort the dependencies alphabetically

Reply:
Will do.